src/Symfony/Component/Form/Form.php

```diff
@@ -416,6 +417,13 @@ public function handleRequest(mixed $request = null): static
         return $this;
     }
 
+    public function isSubmittedRequestValid(Request $request): bool
```

Comment:
you should keep it as optional (e.i. `?Request $request = null`) as `NativeRequestHandler` will expect a `null` value always